dogpile/cache/util.py

```diff
@@ -4,7 +4,7 @@
 from ..util import langhelpers
 
 
-def function_key_generator(namespace, fn, to_str=str):
+def function_key_generator(namespace, fn, to_str=str, use_qual_name=False):
```

Comment:
how about also in:

*  `def function_multi_key_generator`
*  `def kwarg_function_key_generator`